sdk/search/azure-search/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Release History
+
+## 11.0.0-beta.1 (Unreleased)
```

Comment:
If you are preparing for this to be the last PR before release you may want to stamp in the date here instead of Unreleased.